### PR TITLE
TES import

### PR DESCRIPTION
**Pull request by buchanae**
Rough draft of pulling Funnel events from Kafka and writing them into Arachne.

### CODE REVIEW

File: import.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+  "github.com/Shopify/sarama"
+  "github.com/golang/protobuf/proto"
+  "github.com/bmeg/arachne/aql"
+  "github.com/ohsu-comp-bio/mortar/events"
+  "github.com/ohsu-comp-bio/tes"
```
Comment (adamstruck) [inline import.go]: you don't need to import this if your including the proto file and generated code in this repo

**Comment by adamstruck** [comment]
Looks good! 👍 

Generally,  my concern is the proliferation of duplication of code that is starting to occur in these repos:

- `github.com/ohsu-comp-bio/funnel`
- `github.com/ohsu-comp-bio/tes`
- `github.com/ohsu-comp-bio/mortar`

I think we need to start thinking carefully how to split up things conceptually and make sure import things whenever possible.

**Comment by buchanae** [comment]
I removed the `tes` package, which was accidentally committed.

As soon as we resolve https://github.com/ohsu-comp-bio/funnel/issues/481 I'll remove some of the events package too.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+proto:
+	protoc \
+		-I ./ \
+    -I $$GOPATH/src/github.com/golang/protobuf/ptypes/struct/ \
+    -I $$GOPATH/src/github.com/golang/protobuf/ptypes/timestamp/ \
+    --go_out=Mtes/tes.proto=github.com/ohsu-comp-bio/tes:. \
```
Comment (adamstruck) [inline Makefile]: this command should be updated now that you removed the tes.proto file

**Comment by buchanae** [comment]
I've added new functionality to this PR, so please review again.

This creates additional vertex types for each task. Still need edges.

**Comment by buchanae** [comment]
Ok, the latest commit is in decent shape:

- Added edges from task to tag, file, etc.
- Cleaned up client, vertex, edge code
- Cleaned up Kafka event reader
- Added dev docs to readme and helpers to Makefile

I didn't put a ton of thought into vertex/edge label names, GID formats, etc. I'm pretty sure I'm diverging from BMEG on that so far. Probably cleanup to be done there. Also, we should have a style guide for those things.

**Comment by buchanae** [comment]
Added a cobra CLI. I'm done adding to this branch now.

**Comment by buchanae** [comment]
@adamstruck requested tests and the basics of a Go project, so I guess I'm not done yet.

Added tidy/lint in the last commit.

**Pull request merged**